Readme update (#6)
cosmos/registry: README.md

# Overview

> This repository holds data chains in the cosmos ecosystem, the [registrar](https://github.com/cosmos/cosmos-registrar) can be used to submit chains for the registry.

Reviewer: ```suggestion
This repository holds data for chains in the cosmos ecosystem, the [registrar](https://github.com/cosmos/cosmos-registrar) can be used to submit chains to the registry.
```

# Motivation

> Conversations in the Interchain UX Working group and [Chain Agnostic Improvement Proposals](https://github.com/ChainAgnostic/CAIPs/issues/27) around multi-chain assets have shown the need for a registry to discover chains and keep basic information up-to-date in order to enable communication with these chains.

Reviewer: ```suggestion
Conversations in the Interchain [UX Working Group](http://hackmd.io/@okwme/ux-wg) and [Chain Agnostic Improvement Proposals](https://github.com/ChainAgnostic/CAIPs/issues/27) around multi-chain assets have shown the need for a registry to discover chains and keep basic information up-to-date in order to enable communication with these chains.
```

# Goal

> Creation and maintain a global chain registry, to applications to discover and communicate with different chains that support the [IBC protocol](https://github.com/cosmos/ics). 

Reviewer: ```suggestion
Create and maintain a global chain registry for applications to discover and communicate with different chains that support the [IBC protocol](https://github.com/cosmos/ics). 
```

> * Allow contributors to take ownership of the data they submitted

Reviewer: ```suggestion
* Allow contributors to take ownership of the data they submit
```

* Provide all information necessary to communicate with each chain
* Introduce state of chains to navigate active/inactive chains easily
* Make the submission process as easy as possible with [cosmos-registrar](https://github.com/cosmos/cosmos-registrar)

> To utilize a github repository is seen as an intermediate solution by parts of the interchain community that could be replaced by a more decentralized approach like a registry chain or a solution that is achored on an existing chain in the future.

Reviewer: ```suggestion
To utilize a github repository is seen as an intermediate solution by parts of the interchain community that could be replaced by a more decentralized approach like a registry chain or a solution that is anchored on an existing chain in the future.
```
